packages/FieldTypes/stories/FiledTypes.showcase.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import L10n from "@paprika/l10n";
+
+import { getStoryName } from "storybook/storyTree";
+import styled, { css } from "styled-components";
+import FieldTypes from "../src";
+
+export default {
+  title: getStoryName("FieldTypes"),
+};
+
+const Container = styled.div(() => {
+  return css`
+    [data-pka-anchor="data-fields-numeric"] {
+      font-size: 32px;
```

Comment:
Nitpick: Should we use stylers or tokens here instead?

Reply:
is a story, its ok :)